resources/artifact-ignores.properties

```
@@ -538,3 +538,8 @@ warnings
 
 # Suppress a release with regression that causes node's computer to be null
 spotinst-2.0.26
+
+# INFRA-2838 - no pom published, causes update center generation to break
+nexus-jenkins-plugin-3.9.20201203-135942.e9f4ebc
+nexus-jenkins-plugin-3.9.20201203-151437.f2f6b16
+nexus-jenkins-plugin-3.9.20201203-120425.a87655e # useless pom
```

Looks like this might not actually be a comment? 😬 

```suggestion
nexus-jenkins-plugin-3.9.20201203-120425.a87655e
```

---

Nvm, we have some of these already in the file.